src/PresentationalComponents/Input/Input.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classnames from 'classnames';
+
+const checkTypes = ['checkbox', 'radio'];
+
+const Input = ({type, className, ...props}) => {
+  const classes = checkTypes.hasOwnProperty(type) ? 'pf-c-check' : 'pf-c-form-control';
+  return (
+    <input {...props}
+      type={type}
+      className={classnames(classes, className)}
+    />
```

Comment:
Needs to assign `checked` attribute for any checked elements in radios and checkboxes

Reply:
`Checked` attribute will be applied from `{...props}` spread so if we don't need to do anything with chacked no need to add it to props.